dev/teamcity/dist-assets-tc

```diff
@@ -11,6 +11,24 @@ set -x
 
 echo "Asset compilation"
 
+cd static/src/javascripts
+
+set +x
```

Comment:
I see that this is done throughout this file and not unique to this PR, but do you know why we wrap most commands in `+x` and `-x`? This toggles the debug flag but I don't understand why we would, for example, wrap a simple echo command with this.

Reply:
i think `+x` toggles it off? then `set -x` toggles it back on after echoing?

Reply:
Any commands between `set +x` and `set -x` are printed out to the terminal, which has it's uses, but I agree with @nicl it seems redundant for `echo` commands which should be printed out anyway.

Reply:
Yeah, if it is really to toggle things off with echo then a custom function is probably better than this repetition all over the place.